src/backend/web/validation.js

```diff
@@ -73,6 +73,13 @@ const queryValidationRules = [
     .isIn(['post', 'author'])
     .withMessage('invalid filter value')
     .bail(),
+
+  query('perPage', 'per_page needs to be empty or an integer').custom(
```

Comment:
Let's also set a max on this of 10. So it can be nothing, or a number between 1 and 10.